Userland/Services/Taskbar/ShutdownDialog.h

```diff
@@ -13,7 +13,12 @@ class ShutdownDialog : public GUI::Dialog {
     C_OBJECT(ShutdownDialog);
 
 public:
-    static Vector<char const*> show();
+    struct Command {
+        StringView executable;
+        Vector<char const*, 2> arguments;
```

Comment:
Can we use an `Array` here?
If that's possible, you can also make `options` `constexpr`.

Reply:
Not really, the options have different argument lengths, so I'd be a little awkward.

Reply:
My bad, I misread that.